src/Makefile

```
@@ -48,7 +48,10 @@ PREFIX?=/usr/local
 INSTALL_BIN=$(PREFIX)/bin
 INSTALL=install
 PKG_CONFIG?=pkg-config
-PYTHON?=python
+
+ifndef PYTHON
+PYTHON := $(shell which python || which python3)
```

```suggestion
PYTHON := $(shell which python3 || which python)
```

Suggest to prefer python 3.